pauthabielf64/pauthabielf64.rst

```diff
@@ -503,21 +540,36 @@ instructions is:
 With the Armv8.3-a extension the autia1716 and br x17 can be combined
 into a single instruction braa x17, x16
 
-When lazy loading is supported the initial contents of the plt.got[n]
-point to the address of the lazy resolver trampoline at
-plt.got[0]. The dynamic linker must sign all of these addresses at
-load time, limiting the faster start-up time advantage of lazy
-binding.
+Recording a signed PLT GOT in the ELF file
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The static linker communicates a signed PLT GOT entry with an AUTH
+variant ``R_AARCH64_AUTH_JUMP_SLOT`` dynamic relocation with the
+signing schema encoded in the contents of the place.
+
+The dynamic tag ``DT_AARCH64_PAC_PLT`` must also be set if the signing
+schema is the same as the existing ABI. We chose to use
+``R_AARCH64_AUTH_JUMP_SLOT`` as well as ``DT_AARCH64_PAC_PLT`` so we
+can choose to vary the PLT GOT signing schema in future releases,
+which we can do by altering the signing schema encoded in the contents
+of the place of the relocation and dropping the ``DT_AARCH64_PAC_PLT``
+tag.
 
-Implications for the ABI
-------------------------
+Section Types
+=============
 
-As the signing schema used for the GOT can affect code generation, and
-there is a use case for both a signed and unsigned GOT the ELF ABI
-defines mechanisms to do both. It is expected that platforms will
-mandate a single choice, although per link-unit choices are
-possible. It is not possible to mix mechanisms within the same
-link-unit as there is at most one GOT entry per symbol.
+The PAuth ABI adds an additional Processor specific section type
+
+.. table:: ELF Section Types
```

Comment:
in the generated pdf the columns of the table are just off, causing the name and variable columns to cut off the text

Reply:
Thanks for pointing that out, I think I've managed to find a way of getting to render correctly by using `` `` quotes.